varsim_multi.py

```diff
@@ -56,36 +58,64 @@ def varsim_multi(reference,
     restricted_dir = os.path.join(out_dir, "restricted")
 
     restricted_reference, restricted_vcfs = gen_restricted_ref_and_vcfs(reference, variant_vcfs, regions, samples, restricted_dir, flank=0, short_contig_names=False)
+    dgv_vcf = None
 
-    merged_bedtool = pybedtools.BedTool(regions).merge() if regions else None
+    if dgv_file:
+        assert sv_insert_seq, "SV insertion sequence file is required."
```

Comment:
This is only required if there are missing insertion sequences. So, log a warning instead of failing here. Or, perhaps, the tool itself could throw an exception if encounter a missing sequence and no insertion sequence file specified.